tests/Resources/ti.geolocation.addontest.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
```

Comment:
I had to change this test to fix linter warnings, so I also rewrote it in es6 syntax.

BTW linter does not allow **ES6+** syntax in tests (only es6), so no `async`/`awiat` in tests (but test app works fine). And current `es6.*.test.js`-files from `mocha-suite` do not pass linter check if you'll put them inside `test/Resources`

>you can do arrow functions to shorten the syntax in these `it`/`describe` methods

Actually in this test you can't use arrow for `describe`, because it will break
```javascript
this.timeout(6e4); // 60 sec
```
which after transpilation will become
```javascript
global.timeout(6e4);
```